multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangastream/WPMangaStreamGenerator.kt

```diff
@@ -33,6 +33,7 @@ class WPMangaStreamGenerator : ThemeSourceGenerator {
         SingleLang("Manga Pro Z", "https://mangaprotm.com", "ar", overrideVersionCode = 1),
         SingleLang("Manga Raw.org", "https://mangaraw.org", "ja", className = "MangaRawOrg", overrideVersionCode = 1),
         SingleLang("MangaIndonesia", "https://mangaindonesia.net", "id"),
+        SingleLang("Mareceh", "https://mareceh.com", "id", isNsfw = true, overrideVersionCode = 10),
```

Comment:
Sorry about noticing it just now, but since the source changed its name, you need to keep the old package name, otherwise users will not receive any update to the existing extension. The `id` change in this case doesn't matter since the URLs are incompatible anyway.

```suggestion
        SingleLang("Mareceh", "https://mareceh.com", "id", isNsfw = true, pkgName = "mangceh", overrideVersionCode = 10),
```

After changing it, move the overrides from the `multisrc/overrides/wpmangastream/mareceh` folder to `multisrc/overrides/wpmangastream/mangceh`.

After this last change, the PR will be good and can be merged 👍🏻

Reply:
Ohh i see
Thanks for the updated 👍🏻
And I finally understood a new lesson from you.